templates/yymmdd-project-proposal.md

```diff
@@ -43,7 +43,7 @@ _In-depth explanation of maintenance plan_
 
 ## Budget
 
-_Budget for project. Funding for CPS grants is available in bnUSD. Funding for ICON Foundation grants is available in ICX or USD_
+_Budget for project. Funding for CPS grants is available in bnUSD (stablecoin). Funding for ICON Foundation grants is available in ICX or USD_
```

Comment:
Link to bnUsd website like this:

```md
... available in [bnUSD (stablecoin)](https://balanced.network/).
```

The text above, via Markdown formatting, appears like this to the reader:

... available in [bnUSD (stablecoin)](https://balanced.network/).